src/ShinyDeploy/Domain/Deployment.php

```diff
@@ -119,9 +119,20 @@ public function deploy(bool $listMode = false): bool
             return false;
         }
 
+        $this->logResponder->log('Checking repository status...');
+
+        if ($this->checkRepoStatusIsCorrupted()) {
```

Comment:
Please rename method according to naming conversions to "repoStatusIsCurrupted".